src/frontend/js/components/SuccessIcon/index.tsx

```diff
@@ -0,0 +1,15 @@
+export const SuccessIcon = () => {
```

Comment:
Icon are actually referencing small icons, I think it would be out of subject. But we can imagine renaming it to `SuccessIllustration` in a separate PR